### DISCUSSION TITLE
Is there any way to create new database on the project based on CLI?

### DISCUSSION THREAD

**Discussion by developer-kikikaikai** [supabase/supabase#3609] [Questions] [answered]
Hi!

Now I want to create new environment same as current one.
And I also want to get those steps automated.

So I wonder if those steps can do on CLI.

(I guess it's difficult because supabase expects to use RLS for security management)

**Comment by silentworks** [comment, marked as answer]
I don't think this is possible with the CLI at the moment, but for managing environments after you have created the database you can use a migration tool. This is how I currently manage multiple environments with Supabase, you can also set up your RLS within your migrations since they are created purely by SQL queries.

**Reply by cloudorbush** [reply]
> I don't think this is possible with the CLI at the moment, but for managing environments after you have created the database you can use a migration tool. This is how I currently manage multiple environments with Supabase, you can also set up your RLS within your migrations since they are created purely by SQL queries.

which migration tool do you recommend?

**Reply by VictorPeralta** [reply]
Here's an approach that could help you get what you need

https://www.slip.so/tutorials/database-migrations-in-supabase-with-migra
